aiohttp/web.py

```diff
@@ -221,8 +222,18 @@ def middlewares(self):
         return self._middlewares
 
     def make_handler(self, **kwargs):
-        return self._handler_factory(
-            self, self.router, loop=self.loop, **kwargs)
+        debug = kwargs.pop('debug', _sentinel)
```

Comment:
By the way should `_sentinel` be named as `sentinel` if we expose/use it outside the `helpers` module?